tests/writeConcern/writeconcern-ctor-002.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+MongoDB\Driver\WriteConcern construction with 64-bit wtimeoutms
+--SKIPIF--
+<?php if (4 !== PHP_INT_SIZE) { die('skip Only for 32-bit platform'); } ?>
```

Comment:
Why does this test require a 32-bit platform? The range of signed 32-bit ints is -2147483648 through 2147483647, and this test is using a literal value of `INT32_MAX + 1`. If anything, I'd expect it to require a 64-bit environment.